### DISCUSSION TITLE
Why kornia.color.rgb2hsv output range not in 0-1?

### DISCUSSION THREAD

**Discussion by NTUYWANG103** [kornia/kornia#2014] [Questions]
Why kornia.color.rgb2hsv output range not in 0-1? some values even greater than 6? (input 0-1 rgb)

**Comment by edgarriba** [comment]
As stated in the docs, the output values are in the range H 0..2pi. S and V are in the range 0..1.

- https://github.com/kornia/kornia/blob/master/kornia/color/hsv.py#L20
- https://www.lifewire.com/what-is-hsv-in-design-1078068
